documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -217,17 +217,21 @@ Enables or disables features and functionality controlled by xref:ref-operator-c
 `STRIMZI_POD_SECURITY_PROVIDER_CLASS`:: Optional.
 Configures the pluggable `PodSecurityProvider` class which can be used to provide the security context configuration for Pods and containers.
 
-[[STRIMZI_LEADER_ELECTION_ENABLED]]
-`STRIMZI_LEADER_ELECTION_ENABLED`:: Optional, default `false`.
-Enables or disables the leader election which allows to run additional operator instances as a hot-stand-by.
-Disabled by default.
+[id='ref-operator-cluster-leader-election-{context}']
+== Leader election environment variables 
+
+Use leader election environment variables when xref:assembly-using-multiple-cluster-operator-replicas-{context}[running additional Cluster Operator replicas].
+You might run additional replicas to safeguard against disruption caused by major failure.
+
+`STRIMZI_LEADER_ELECTION_ENABLED`:: Optional, disabled (`false`) by default.
+Enables or disables leader election that allows additional Cluster Operator replicas to run on standby.
```

Comment:
Should we clarify that it is disabled by default when the environment variable is set, but that the installation files actually enable it?

Reply:
I've added this note. Not sure if it's clear enough:

> NOTE: Leader election is disabled by default.
> It is only enabled when applying this environment variable on installation.